bokeh/settings.py

```diff
@@ -96,7 +96,7 @@
 
 API
 ~~~
-
+d
```

Comment:
This seems like a mistaken add?

Reply:
Not sure how that got in there, but it should be good now.